scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -116,6 +123,19 @@
     default="agent_syslog.log",
 )
 
+define_config_option(
+    __monitor__,
+    "message_log_template",
```

Comment:
Since we do not bundle the `syslogmp` source with agent, I think we should mention here, that this option can be used only in dockerized agent, and not agent that is installed from packages.

Reply:
Will do, I didn't like introducing a dependency but implementing the parsing myself would've been too time consuming

Reply:
Yeah, I can totally understand that...
So I think it should be enough just to be sure that the `syslog_monitor` still can operate without `syslogmp` and just explicitly state here, that new config is not for packages agent.

Reply:
Yep, verified that the monitor works fine without syslogmp if the new option message_log_template is not used